src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -243,6 +243,23 @@ namespace Microsoft.FSharp.Collections
         [<CompiledName("Empty")>]
         val empty<'T> : seq<'T>
 
+        /// <summary>Produces the set difference of two sequences by using generic hash and equality comparisons to compare values.</summary>
```

Comment:
I thinks it#s exactly the point of https://github.com/Microsoft/visualfsharp/pull/253/files#r25061210